hello/views.py

```diff
@@ -1,5 +1,19 @@
-from django.shortcuts import render
+from django.shortcuts import render , redirect
+from .forms import DealerForm
 from django.http import HttpResponse
 
+
 def home(request):
-    return HttpResponse("yo")
\ No newline at end of file
+    return HttpResponse("yo")
+def showformdata(request):
```

Comment:
This view is displaying form for `Dealer` model and creating its instance in db. So, this view name should be `CreateDealerView`.